msteams-platform/concepts/design/design-teams-app-basic-ui-components.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Design your app with basic Fluent UI components
 author: heath-hamilton
-description: Learn the UI components used to build Microsoft Teams apps and get the Teams UI Kit. Learn about fluent kit breadcrumb button card carousel dropdown coachmark picker pivot toggle tooltip
+description: Learn how to use UI components from Teams UI kit to build Microsoft Teams apps. Learn to use UI components such a Fluent kit, breadcrumb, button card, carousel, dropdown, and toggle.
```

Comment:
Fluent is not a kit and it is not a UI component. 
A generic feedback: When we want to mention proper nouns and branded words, it helps to do a quick Google search and read about what those term mean.

Reply:
Edited and updated the description as suggested.